docs/api/web-contents-view.md

```diff
@@ -0,0 +1,123 @@
+# WebContentsView
+
+> A View that displays a WebContents.
+
+Process: [Main](../glossary.md#main-process)
+
+This module cannot be used until the `ready` event of the `app`
+module is emitted.
+
+```javascript
+const win = new BaseWindow({ width: 800, height: 400 })
+
+const view1 = new WebContentsView()
+win.contentView.addChild(view1)
+view1.webContents.loadURL('https://my.app/one')
+view1.setBounds({ x: 0, y: 0, width: 400, height: 400 })
+
+const view2 = new WebContentsView()
+win.contentView.addChild(view2)
+view2.webContents.loadURL('https://my.app/two')
+view1.setBounds({ x: 400, y: 0, width: 400, height: 400 })
```

Comment:
Should `addChild` here be `addChildView` (changed below)? I also think the last setBounds call should be on view2:

```suggestion
const view1 = new WebContentsView()
win.contentView.addChildView(view1)
view1.webContents.loadURL('https://my.app/one')
view1.setBounds({ x: 0, y: 0, width: 400, height: 400 })

const view2 = new WebContentsView()
win.contentView.addChildView(view2)
view2.webContents.loadURL('https://my.app/two')
view2.setBounds({ x: 400, y: 0, width: 400, height: 400 })
```

Reply:
Ah, I remember what I was thinking when I wrote this—that perhaps `addChildView` should instead be named `addChild`. I'm going to give into inertia here though and leave it named `addChildView` as that matches the underlying Views name for the function. I'll fix the docs with your suggestion.